src/radical/pilot/agent/lm/base.py

```diff
@@ -170,10 +174,10 @@ def lrms_config_hook(cls, name, cfg, lrms, logger, profiler):
         }.get(name)
 
         if not impl:
-            logger.info('no LRMS config hook defined for LaunchMethod %s' % name)
+            logger.info('no config hook defined for LaunchMethod %s' % name)
```

Comment:
I think that the previous log entry was a bit more clear. But, it's your decision